docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/chip/chip.event.js

```diff
@@ -0,0 +1,13 @@
+// phpcs:ignoreFile
```

Comment:
please rename to `chip.stories.event.js` to distinguish from actual component JS